modules/ossm-jaeger-architecture.adoc

```diff
@@ -0,0 +1,21 @@
+////
+This CONCEPT module included in the following assemblies:
+-ossm-jaeger.adoc
+////
+
+[id="ossm-jaeger-architecture_{context}"]
+= Jaeger architecture
+
+Jaeger is made up of several components that work together to collect, store, and display tracing data.
+
+* *Jaeger Client* (Tracer, Reporter, instrumented application, client libraries)- Jaeger clients are language specific implementations of the OpenTracing API. They can be used to instrument applications for distributed tracing either manually or with a variety of existing open source frameworks, such as Camel (Fuse), Spring Boot (RHOAR), MicroProfile (RHOAR/Thorntail), EAP (Wildfly), and many more, that are already integrated with OpenTracing.
```

Comment:
Sorry, just realised that in the framework list is done as "open source name (product)" - except "EAP (Wildfly)" - so that should change to "Wildfly (EAP)".

Reply:
It was my fault, that how I described it :)